simplex/storage.go

```diff
@@ -36,8 +36,8 @@ var (
 )
 
 type Storage struct {
-	// height represents the number of blocks indexed in storage.
-	height atomic.Uint64
+	// numBlocks represents the number of blocks indexed in storage.
```

Comment:
// numBlocks represents the number of blocks indexed in storage, also known as the height of the chain